src/metabase/legacy_mbql/schema.cljc

```diff
@@ -925,14 +925,17 @@
 (defclause ^{:requires-features #{:basic-aggregations}} case
   clauses CaseClauses, options (optional CaseOptions))
 
+(defclause ^:sugar ^{:requires-features #{:basic-aggregations}} [case:if if]
```

Comment:
Note - I didn't manage to make it work with just `if`. `clojure.core/if` (or `core/if`, given the namespace alias) doesn't work with kondo. But maybe it's even better this way.

Reply:
`if` is not a `clojure.core` var like `+`. It's not a function, nor a macro, it's a _special form_ implemented specially by the compiler. So we can't play games like `:exclude` on it.